gb-cpu/src/memory/mod.rs

```diff
@@ -0,0 +1,76 @@
+mod zone;
+
+use crate::error::Error;
+
+use crate::address;
+use crate::address::area::Area;
+use crate::address::consts::*;
+use zone::wram::Wram;
+
+#[derive(Debug, Default)]
+pub struct Memory {
+    pub wram: Wram,
+}
+
+impl Memory {
```

Comment:
Duplication of `Address_bus`

Reply:
It's not a duplicate, it's a re-implementation of a single point of access to interact with different types of memories.

Each member of this struct is a dedicated memory manager, it's easier to reason about for me.

Reply:
Again don't reimplement something that already exist. Doing that let me think the work done by other people is not useful for you

Reply:
On the contrary, if i think something could be improve it's my responsibility to propose something.

It's nothing against you, it's just about trying to make the best possible work. I expect you to critique my work as ruthlessly as possible so i can improve, and I'll do the same, for the quality of the work 😄.

Reply:
You could directly rewrite what's already defined instead of creating a new type

Reply:
I don't want to modify without we both agree about the change and i feel more comfortable having a parallel proposition in parallel , and combine the final result for the PR.